packages/db/migrations/0029_warm_blackheart.sql

```
@@ -0,0 +1,16 @@
+	"id" text PRIMARY KEY NOT NULL,
+	"owner_user_id" text NOT NULL,
+	"thread_id" text NOT NULL,
+	"tool_call_id" text NOT NULL,
+	"run_key" text NOT NULL,
+	"analysis" text NOT NULL,
+	"title" text NOT NULL,
+	"blocks" jsonb NOT NULL,
+	"created_at" bigint NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "analysis_result_blocks" ADD CONSTRAINT "analysis_result_blocks_owner_user_id_users_owner_user_id_fk" FOREIGN KEY ("owner_user_id") REFERENCES "public"."users"("owner_user_id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)